e2e/atlas/helper_test.go

```diff
@@ -240,6 +242,18 @@ func RandProjectNameWithPrefix(prefix string) (string, error) {
 	return prefixedName, nil
 }
 
+func MongoDBMajorVersion() (string, error) {
+	atlasClient := mongodbatlas.NewClient(nil)
+	atlasURL := os.Getenv("MCLI_OPS_MANAGER_URL")
+	atlasClient.BaseURL, _ = url.Parse(atlasURL)
+	version, _, err := atlasClient.DefaultMongoDBMajorVersion.Get(context.TODO())
```

Comment:
let's use background instead of TODO
```suggestion
	version, _, err := atlasClient.DefaultMongoDBMajorVersion.Get(context.Background())
```